From e4d51884e3ca36b8256c33936dc31e77e0ad4736 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 8 May 2018 12:35:43 -0400
Subject: [PATCH] Ticket 49649

Description:  Fix crpyt.h include

https://pagure.io/389-ds-base/issue/49649

Reviewed by: mreynolds(one line commit rule)

(cherry picked from commit 2817f0c49401056835a79aafd8f8d4edc9113d1d)
---
 ldap/servers/plugins/pwdstorage/crypt_pwd.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/ldap/servers/plugins/pwdstorage/crypt_pwd.c b/ldap/servers/plugins/pwdstorage/crypt_pwd.c
index 0dccd1b51..19894bd80 100644
--- a/ldap/servers/plugins/pwdstorage/crypt_pwd.c
+++ b/ldap/servers/plugins/pwdstorage/crypt_pwd.c
@@ -20,15 +20,7 @@
 #include <string.h>
 #include <sys/types.h>
 #include <sys/socket.h>
-#if defined(hpux) || defined(LINUX) || defined(__FreeBSD__)
-#ifndef __USE_XOPEN
-#define __USE_XOPEN /* linux */
-#endif              /* __USE_XOPEN */
-#include <unistd.h>
-#else /* hpux */
-#include <crypt.h>
-#endif /* hpux */
-
+#include <crypt.h>  /* for crypt_r */
 #include "pwdstorage.h"
 
 static PRLock *cryptlock = NULL; /* Some implementations of crypt are not thread safe.  ie. ours & Irix */
-- 
2.13.6

